CHANGELOG.next.asciidoc

```diff
@@ -107,6 +107,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...main[Check the HEAD dif
 - Add `application/zip` decoder to the `httpsjon` input. {issue}31282[31282] {pull}31304[31304]
 - Sanitize the Azure storage account container names with underscores (_). {pull}31384[31384]
 - Add missing docs for the `delegated_account` option in the `httpjson` input. {pull}31498[31498]
+- Registry flush time defaults to 1s. {pull}31473[31473]
```

Comment:
We should explain why we did this in the CHANGELOG, and perhaps document what we changed the default value from so users know how to get the old behaviour back.

Reply:
Done on 0b2896b. Does it looks good? Should I add something else?